pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2276,6 +2276,11 @@
         doc = "If true, export topic level metrics otherwise namespace level"
     )
     private boolean exposeTopicLevelMetricsInPrometheus = true;
+    @FieldContext(
+            category = CATEGORY_METRICS,
+            doc = "If true, export buffered broker metrics"
+    )
+    private boolean exposeBufferedBrokerMetrics = false;
```

Comment:
what about "bufferMetrics" ?

this change also will apply to Proxy and Function Worker, so I would drop the word "broker"

Reply:
Thanks for your review. Resolved